lib/ansible/modules/crypto/openssl_publickey.py

```diff
@@ -116,19 +130,35 @@ def __init__(self, module):
         self.name = os.path.basename(module.params['path'])
         self.path = module.params['path']
         self.privatekey_path = module.params['privatekey_path']
+        self.privatekey = None
         self.changed = True
+        self.fingerprint = {}
         self.check_mode = module.check_mode
 
 
+    def get_fingerprint(self):
```

Comment:
Still concerned about the duplicate method (same code in `openssl_publickey` and `openssl_privatekey`).